src/main/resources/db/migration/schema/V20210203154851__course_migration.sql

```diff
@@ -0,0 +1,6 @@
+CREATE TABLE public.registration_course (
+                                            id serial NOT NULL,
```

Comment:
id int8 NOT NULL DEFAULT nextval('nome_do_seu_sequenciador'), ver como criar sequenciador: https://github.com/EmersonDantas/S-W-R-Social-Network-API/blob/master/src/main/resources/db/migration/schema/V20200627152100__create_rebel.sql